Automattic/pocket-casts-android
# Transcript - Transition from player screen

## Description
When the transcript shelf item is clicked, a blank transcript screen opens with an animation.

** I've tried to match [animation on iOS](https://github.com/Automattic/pocket-casts-ios/pull/1899).

## Testing Instructions
1. Play an episode from "Cautionary Tales" (https://pcast.pocketcasts.net/8zpmjiru)
2. Open full screen player
3. Tap on the Transcript shelf icon
4. ✅ Notice that a blank Transcript screen is shown with a transition, player controls are shown at the bottom
5. Drag up on the Transcript screen
6. ✅ Notice that the up next screen is not shown
7. Tap on the close button on Transcript screen
8. ✅ Notice that Transcript screen is closed
9. Drag up on the Player screen
10. ✅ Notice that the up next screen is shown

## Screenshots or Screencast 

Portrait - Phone

https://github.com/user-attachments/assets/62ee69ea-3406-4441-8470-5919b6a3999b

Landscape - Phone

https://github.com/user-attachments/assets/e410fa50-6f0b-4117-a19a-c449be8d1d36

** I need to check the transition for the Tablet landscape. On iOS, the iPad landscape layout is the same as the portrait. We might want to copy that behavior.

## Checklist
 
#### I have tested any UI changes...
